tembo-cli/src/cmd/apply.rs

```diff
@@ -762,17 +766,21 @@ pub fn get_rendered_dockerfile(
 }
 
 fn get_postgres_config(
+    extensions: Option<&Vec<ControllerExtension>>,
     instance_ps_config: Option<HashMap<String, Value>>,
     postgres_configs: &std::option::Option<Vec<ControllerPgConfig>>,
 ) -> String {
     let mut postgres_config = String::from("");
-    let mut shared_preload_libraries: Vec<String> = Vec::new();
+    let mut shared_preload_libraries: Vec<Library> = Vec::new();
 
     if let Some(ps_config) = postgres_configs {
         for p_config in ps_config.clone().into_iter() {
             match p_config.name.as_str() {
                 "shared_preload_libraries" => {
-                    shared_preload_libraries.push(p_config.value.to_string());
+                    shared_preload_libraries.push(Library {
+                        name: p_config.value.to_string(),
+                        priority: MAX_INT32,
```

Comment:
I don't think there is any way to get priority for these so setting them to `2147483647`.

Reply:
This is when `shared_preload_libraries` is specified in stack, app or directly for an instance.